src/posix/platform/system.c

```diff
@@ -150,6 +150,7 @@ otInstance *otSysInit(int aArgCount, char *aArgVector[])
 #if OPENTHREAD_POSIX_VIRTUAL_TIME
     otSimInit();
 #endif
+    platformSettingsInit();
```

Comment:
should be called after platformRadioInit()

Reply:
Updated